xarm_description/urdf/xarm7.urdf.xacro

```diff
@@ -442,7 +442,7 @@
         rpy="0 0 0" />
       <geometry>
         <mesh
-          filename="package://xarm_description/meshes/collision/end_tool.STL" />
+          filename="package://xarm_description/meshes/xarm7/collision/link7.STL" />

```

Comment:
Is there a particular reason why the filename was changed?

Reply:
https://github.com/xArm-Developer/xarm_ros/issues/97
I created this mesh from visual/link7.STL

Reply:
Make sense.
Based on the given issue, I assumed that the shared `end_tool` stl file does not entirely reflect the other xarm series hence the change in filename.